examples/routingServer.ts

```diff
@@ -46,6 +46,9 @@ router
     } else {
       return notFound(context);
     }
+  })
+  .post("/", async (context) => {
```

Comment:
Why are you adding code here?

Reply:
I want to make sure people know how to write multiple methods and handle requests